# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-align/blocks/align-content-block-overflow-000.html.ini
# Commit: b0b878dae244
# Full Hash: b0b878dae244416c3b4799f43ee2f5176f3d3fc8
# Author: fantasai <fantasai.cvs@inkedblade.net>
# Date: 2024-02-28 04:30:28
# Regressor Bug: 1765615
# File Overlap Count: 1
# Description:
#   Bug 1684236 - Implement 'align-content' on block containers. r=layout-reviewers,jfkthame
#   
#   This is Fantasai's original patch, massively simplified:
#   
#    * We now can switch whether we're a BFC dynamically (bug 1765615), which
# ==============================================================================

diff -r 6b8582441a76 -r b0b878dae244 testing/web-platform/meta/css/css-align/blocks/align-content-block-overflow-000.html.ini
--- a/testing/web-platform/meta/css/css-align/blocks/align-content-block-overflow-000.html.ini	Tue Feb 27 11:35:54 2024 -0500
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,2 +0,0 @@
-[align-content-block-overflow-000.html]
-  expected: FAIL